satellite/admin/user.go

```diff
@@ -0,0 +1,168 @@
+// Copyright (C) 2020 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package admin
+
+import (
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"golang.org/x/crypto/bcrypt"
+
+	"storj.io/common/uuid"
+	"storj.io/storj/satellite/console"
+)
+
+func (server *Server) addUser(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	body, err := ioutil.ReadAll(r.Body)
```

Comment:
I think we need to call close for `r.Body`

Reply:
Good call, lemme add that!

Reply:
according to the docs, the method does that automatically?

Reply:
https://golang.org/pkg/io/ioutil/#ReadAll I don't see where it automatically close the body?

Reply:
@egonelbre do you know?

Reply:
See https://golang.org/pkg/net/http/#Request.Body